pkg/devfile/parser/context/context.go

```diff
@@ -46,12 +44,6 @@ func (d *DevfileCtx) populateDevfile() (err error) {
 		return err
 	}
 
-	// Check if the apiVersion is supported
-	if !d.IsApiVersionSupported() {
```

Comment:
The function `IsApiVersionSupported()` is now not used anywhere. Should we remove it?

Reply:
I kept it just in case we wanted to use it but I certainly can remove it.

Reply:
Ok let's keep it for now